documentation/plain_rules.md

```diff
@@ -124,7 +124,7 @@ Sends an email to the recipient set in the action parameters, with the body mail
     }
 ```
 
-The `template` field performs [string substitution](#string-substitution-syntax).
+The `template`, `from`, `to` and `subject` fields perform [string substitution](#string-substitution-syntax).
```

Comment:
I understand that to keep documentation coherent, the list just under "This substitution can be used in the the following fields" at "String substitution syntax" should be also modified.